packages/@sanity/code-input/src/Schema.js

```diff
@@ -0,0 +1,25 @@
+export default {
+  name: 'code',
+  type: 'object',
+  fields: [
+    {
+      title: 'Code',
+      name: 'code',
+      type: 'text'
+    },
+    {
+      name: 'language',
+      title: 'Language',
+      type: 'string'
+    },
+    {
+      title: 'Highlighted lines',
+      name: 'highlightedLines',
+      type: 'array',
+      of: [{
+        type: 'string',
+        title: 'Highlighted line'
```

Comment:
Looking at it, I think its just a matter of converting the line numbers from strings to numbers before emitting the set patch